packages/circuit-ui/components/NotificationToast/NotificationToast.stories.tsx

```diff
@@ -87,6 +83,32 @@ Base.play = async ({ canvasElement }: { canvasElement: HTMLCanvasElement }) => {
   await screen.findByRole('status');
 };
 
+export const Base = (toast: NotificationToastProps): JSX.Element => (
+  <ToastProvider>
+    <App toast={toast} />
+  </ToastProvider>
+);
+
+Base.play = play;
+
+export const WithTopPosition = (toast: NotificationToastProps): JSX.Element => (
+  <ToastProvider position="top">
+    <App toast={toast} />
+  </ToastProvider>
+);
+
+WithTopPosition.play = play;
+
+export const WithTopRightPosition = (
+  toast: NotificationToastProps,
+): JSX.Element => (
+  <ToastProvider position="top-right">
+    <App toast={toast} />
+  </ToastProvider>
+);
+
+WithTopRightPosition.play = play;
```

Comment:
We use chromatic.com to catch visual bugs by taking a screenshot of every story. We're billed per story, so we try to combine related stories into one to save costs.

Could you please merge these into a single "Position" story?